quicklisp/client.lisp

```diff
@@ -48,6 +48,9 @@
 (defun system-list ()
   (provided-systems t))
 
+(defun all-installed-systems ()
+  (filter #'installedp (system-list)))
```

Comment:
I don't think FILTER is a good utility function name. I'd prefer to stick with the built-in REMOVE-IF-NOT (which is also not a great name).